client/src/components/Ticket.js

```diff
@@ -0,0 +1,110 @@
+import React, { useState, useEffect } from 'react';
+import './Ticket.css';
+import ReactCardFlip from 'react-card-flip'; // comopnent that will flip the ticket
+
+function Ticket(props) {
+  const [isFlipped, setIsFlipped] = useState(true); // state for flipping animation
+  const [contentClass, setContentClass] = useState('content'); // state for changing the content class name
+  const [display, setDisplay] = useState('flex'); // state for hidding and displaying the ticket
+  const [buttonDisplay, setButtonDisplay] = useState('none'); // state for hidding and displaying the hide button
+  const [classTicket, setClassTicket] = useState('ticket'); // state for changing the ticket class name
+  // function to add zeros to the date if needed
+  function addZero(i) {
+    if (i < 10) {
+      i = `0${i}`;
+    }
+    return i;
+  }
+  // function to generate the current date
+  function myFunction() {
```

Comment:
`yourFunction` is not a great name. name it after what it does.
also, if it not very specific for the inner logic of the `Ticket` component, no need to define it inside it.

Reply:
Fixed .